shapeshift/web
# fix: use loading to determine if asset should show circular progress

… or double dash for assets without asset data

## Description

Now using a loading state for market data to differentiate an asset on the dashboard that is loading vs an asset for which no asset data exists.

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)

closes #375

## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps
2. Load dashboard. Yearn vault assets with a balance should show on the dashboard. Since there is no market data, it should show '--' instead of a circular loader.

## Screenshots (if applicable)

<img width="986" alt="Screen Shot 2021-11-15 at 3 58 41 PM" src="https://user-images.githubusercontent.com/88169003/141867256-45a702a2-1261-4455-ad9a-17307daecdd7.png">
